lms/js_config_types.py

```diff
@@ -0,0 +1,27 @@
+from typing import NotRequired, TypedDict
+
+
+class APICallInfo(TypedDict):
+    path: str
+    authUrl: NotRequired[str]
+
+
+class APIAssignment(TypedDict):
+    id: int
+    title: str
+
+
+class APIAssignmentStats(TypedDict):
+    display_name: str
```

Comment:
Should this be `title`? `display_name` makes sense for users, as they have other kinds of name (username, formal name etc.)

Reply:
No, the issue is the naming of the class, it should be StudentStats and the method should be typed as returning a list instead. 

I'll fix it.